sklearn/tests/test_multioutput.py

```diff
@@ -336,3 +341,123 @@ def test_multi_output_exceptions():
     y_new = np.column_stack((y1, y2))
     moc.fit(X, y)
     assert_raises(ValueError, moc.score, X, y_new)
+
+
+def test_classifier_chain_fit_and_predict():
+    """Fit classifier chain and verify predict performance"""
+    X, Y = datasets.make_multilabel_classification(n_samples=10000,
```

Comment:
Note that `make_multilabel_classification` is not a very good dataset for classifier chains: there is no covariance between labels imposed by the model.